main.js

```diff
@@ -1,49 +1,187 @@
-/*task 6*/
-let user = 'John Doe';
-let student = 'Irina';
-
-console.log(user);
-console.log(student);
-
-user = student; /*Irina*/
-
-console.log(user);
-
-/*task 7*/
-let test = 1;
-
-test++; /*2*/
-test = test + '1'; /*21*/
-
-console.log(test);
-
-test = test - 1; /*20*/
-
-console.log(test);
-
-test = Boolean(test);
-
-console.log(test);
-
-/*task 8*/
-let arr = [2, 3, 5, 8];
-let result = 1;
-
-for (let i = 0; i < arr.length; i++) {
-    result *= arr[i];
+//task 19.5
+const palindrome = (word) => word.charAt(0) === word.charAt(word.length - 1);
```

Comment:
What about the words with more than 3 chars?

Reply:
I maked this task correctly just now.